packages/components/src/Modal/Modal.css

```diff
@@ -81,7 +81,7 @@
 
 :global(.jobber-retheme) .header h3 {
   font-family: var(--typography--fontFamily-normal);
-  font-size: var(--typography--fontSize-largest);
+  font-size: var(--typography--fontSize-larger);
```

Comment:
Just delete this declaration and replace the Heading in the Modal tsx with level 2!

Reply:
I have this done [here](https://github.com/GetJobber/atlantis/pull/1840/files#diff-2a52591e6f55264bf76c5b2f7ace34e83339e786010d5ec3800069f2208e8f3cR117) if you want to reference

Reply:
H2 would have a larger font size, however I don't think this is needed anymore since styling of the base header should match. I am going to verify quickly before pushing the change

Reply:
Based on this I think that should be a level 2
<img width="534" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/f9a6cf61-1553-4bf0-a4e8-8c365f11a589">

Reply:
Yes so I think we want `Heading level={2}` to be used here, instead of specifying a `Heading level={3}` and then injecting styling here to make it _look like_ a `Heading level={2}`

Reply:
I see, I will test my change locally and push it up so semantically it will be an H2 with H3 styling

Reply:
Sounds good, I will update it